### DISCUSSION TITLE
Traversing though Information in a tweepy.models.Status Object

### DISCUSSION THREAD

**Discussion by jai-dewani** [tweepy/tweepy#1523] [Questions] [answered]
**Problem:**  
When wanting to extract specific data out of a series of tweets, one has to understand how the `tweepy.model.Status` class has all the information stored but I couldn't find an easy way to find all the properties that are in an object of this class, or is there a way to pretty print this class because it can have a lot on information and going thought it can be a very tedious task.

**My Solution:**
allow users to call a `.key()` method like python dictionaries to see the properties the object has   
or 
Allow users to pretty print the object which can greatly increase the readability of the object

**Comment by Harmon758** [comment, marked as answer]
Tweepy's Status objects are a representation of the [Tweet objects](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet) Twitter's API returns.
The documentation for Twitter's API includes descriptions for each attribute.

You can use [Python's built-in `dir` function](https://docs.python.org/3/library/functions.html#dir) to list valid attributes for an object.

You can also retrieve the raw JSON returned by Twitter's API by using the `_json` attribute of a Status object  and “pretty-print” it using [Python's pprint standard library module](https://docs.python.org/3/library/pprint.html).
